### PR TITLE
[API] Embed full Manufacturer object in AssetModel.

### PR DESCRIPTION
**Pull request by xor-xor**

### CODE REVIEW

**Comment by szok** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

File: src/ralph/assets/api/views.py [hunk 1]
```diff
@@ -81,6 +81,8 @@ class CategoryViewSet(RalphAPIViewSet):
 class AssetModelViewSet(RalphAPIViewSet):
     queryset = models.AssetModel.objects.all()
     serializer_class = serializers.AssetModelSerializer
+    save_serializer_class = serializers.AssetModelSaveSerializer
+    prefetch_related = ['manufacturer']
```
Comment (ar4s) [inline src/ralph/assets/api/views.py]: `select_related` instead of `prefetch_related`

**Comment by ar4s** [comment]
:ok:

**Pull request merged**